static/contact.html

```diff
@@ -56,8 +58,8 @@ <h2 id="community">
             </h2>
             <p>The main forum currently used to discuss the project is the official
             <a href="https://reddit.com/r/GrapheneOS">/r/GrapheneOS subreddit</a> on Reddit.</p>
-            <p>The official IRC channel is #grapheneos on irc.freenode.net. The channel is bridged
-            to Matrix and is available at #grapheneos:matrix.org for Matrix users.</p>
+            <p>The official IRC channel is <code>#grapheneos</code> on <code>irc.freenode.net</code>. The channel is bridged
```

Comment:
I'm not sure that I want to change how this is styled. It probably belongs in a separate pull request either way.